README.md

```diff
@@ -1,3 +1,125 @@
-# wallet-backend
+# Wallet-Backend
 
-A backend for Stellar wallet applications
+A backend for Stellar wallet applications that provides transaction submission, account management, and 
+payment tracking capabilities. 
+
+## Overview
+
+The wallet-backend service provides several key functionalities:
+
+- Account sponsorship and creation
+- Transaction submission and tracking
+- Payment history tracking
+- Fee bump transaction support
+- Channel account management
+- Transaction submission service (TSS)
+
+## Local Development Setup
+In this section, we will go through the steps required to start the wallet-backend server for local development
+and contribution.
+
+### Prerequisites
+
+- Go 1.23.2 or later
+- Docker and Docker Compose
+
+### Running the Server
+
+1. Clone the repository:
+```bash
+git clone https://github.com/stellar/wallet-backend.git
+cd wallet-backend
+```
+
+2. The wallet-backend spins up different services through its docker-compose file. There are two ways to start the
+server:
+
+#### Docker
+This is the simplest and quickest way to start wallet-backend server. All the relevant services will be started as
+docker services in their respective containers. This config is defined in `docker-compose.yaml` file in the main directory.
+
+1. Copy the example `.env.example`:
+```bash
+cp .env.example .env
+```
+
+2. Add the environment variables:
+```bash
+CHANNEL_ACCOUNT_ENCRYPTION_PASSPHRASE=
+DATABASE_URL=postgres://postgres@localhost:5432/wallet-backend?sslmode\=disable
+DISTRIBUTION_ACCOUNT_PRIVATE_KEY=
+DISTRIBUTION_ACCOUNT_PUBLIC_KEY=
+LOG_LEVEL=TRACE
+NETWORK=testnet
+PORT=8001
+RPC_URL=http://localhost:8000
+SERVER_BASE_URL=http://localhost:8001
+STELLAR_ENVIRONMENT=development
+TRACKER_DSN=
+WALLET_SIGNING_KEY=
+```
```

Comment:
can you also add the variables listed in tss_options.go ? These are used to configure the channels that TSS uses.